draft-ietf-elegy-rfc8989bis.md

```diff
@@ -210,15 +216,44 @@ Assuming that 300 legitimate volunteers are all from different organizations,
 three conspiring organizations would need 771 volunteers (257 per organization)
 for a 50% chance of NomCom capture (see {{capture-math}}).
 
-## One Year of Participation
+### One Year of Participation
 
 Attendance at 3 meetings requires at least 8 months. Given the volume of
 volunteers necessary to capture the process, an attack requires a surge in
-attendees over the course of a year. IETF Secretariat SHOULD analyze unexplained
-surges in attendance to look for signs of manipulating the eligibility
-requirements (e.g. logging in to a single session and then immediately logging
-out). In the event of abuse of process, the leadership would then have months to
-adjust policy in response before the NomCom cycle begins.
+attendees over the course of a year. The IETF Secretariat SHOULD analyze
+unexplained surges in attendance to look for signs of manipulating the
+eligibility requirements (e.g. logging in to a single session and then
+immediately logging out). In the event of abuse of process, the leadership would
+then have months to adjust policy in response before the NomCom cycle begins.
```

Comment:
We should not task the secretariat with this, since they don't know what an "unexplained surge" might look like.

If we want a safeguard here, it should IMO take the form of a community review/challenge period.